ui/public/locales/pt_BR.json

```diff
@@ -1404,146 +1836,191 @@
+"label.scheduled.snapshots": "Snapshots programados",
 "label.scope": "Escopo",
 "label.search": "Pesquisar",
-"label.secondary.staging.store": "Armazenamento de Est\u00e1gio Secund\u00e1rio",
-"label.secondary.staging.store.details": "Detalhes do Armazenamento de Est\u00e1gio Secund\u00e1rio",
-"label.secondary.storage": "Storage Secund\u00e1rio",
-"label.secondary.storage.count": "Pools de Storage secund\u00e1rios",
-"label.secondary.storage.details": "Detalhes de armazenamento secund\u00e1rio",
-"label.secondary.storage.vm": "VM de storage secund\u00e1rio",
-"label.secondary.used": "Uso do Storage Secund\u00c3\u00a1rio",
+"label.secondary.isolated.vlan.type.isolated": "Isolada",
+"label.secondary.isolated.vlan.type.promiscuous": "Prom\u00edscuos",
+"label.secondary.staging.store": "Armazenamento de est\u00e1gio secund\u00e1rio",
+"label.secondary.staging.store.details": "Detalhes do armazenamento de est\u00e1gio secund\u00e1rio",
+"label.secondary.storage": "Armazenamento secund\u00e1rio",
+"label.secondary.storage.count": "Pools de armazenamento secund\u00e1rios",
+"label.secondary.storage.details": "Detalhes do armazenamento secund\u00e1rio",
+"label.secondary.storage.vm": "VM de armazenamento secund\u00e1rio",
+"label.secondary.used": "Uso do armazenamento secund\u00c3\u00a1rio",
 "label.secondaryips": "IPs secund\u00e1rios",
-"label.secondarystoragelimit": "Limites do Storage Secund\u00e1rio (GiB)",
-"label.secretkey": "Chave Secreta",
-"label.security.group.name": "Nome do Security Group",
+"label.secondarystoragelimit": "Limites do armazenamento secund\u00e1rio (GiB)",
+"label.secretkey": "Chave secreta",
+"label.secured": "Protegido",
+"label.security.group.name": "Nome do grupo de seguran\u00e7a",
 "label.security.groups": "Grupos de seguran\u00e7a",
-"label.security.groups.enabled": "Security Groups Ativado",
-"label.securitygroup": "Security Group",
+"label.security.groups.enabled": "Grupos de seguran\u00e7a ativado",
+"label.securitygroup": "Grupo de seguran\u00e7a",
+"label.securitygroupenabled": "Grupo de seguran\u00e7a ativado",
 "label.securitygroups": "Grupos de seguran\u00e7a",
 "label.select": "Selecionar",
 "label.select-view": "Selecionar visualiza\u00e7\u00e3o",
 "label.select.a.zone": "Selecione uma zona",
+"label.select.deployment.infrastructure": "Selecione uma infraestrutura de implanta\u00e7\u00e3o",
+"label.select.host": "Selecione o host",
 "label.select.instance": "Selecionar inst\u00e2ncia",
 "label.select.instance.to.attach.volume.to": "Escolha uma inst\u00e2ncia para conectar o volume",
 "label.select.iso.or.template": "Selecione ISO ou template",
-"label.select.offering": "Selecionar Oferta",
-"label.select.project": "Selecionar Projeto",
-"label.select.region": "Selecione Regi\u00e3o",
+"label.select.offering": "Selecionar oferta",
+"label.select.project": "Selecionar projeto",
+"label.select.projects": "Selecionar projetos",
+"label.select.region": "Selecione regi\u00e3o",
```

Comment:
I am not sure if this is a "**Selecione** região" or a "**Selecionar** região".
I would go with `Selecionar região.

I could very well be out of context here (long ago since I don't use the UI and it is hard sometimes to link the label with the UI context).